lib/ex_webrtc/media/ivf_reader.ex

```diff
@@ -0,0 +1,139 @@
+defmodule ExWebRTC.Media.IVFHeader do
+  @moduledoc """
+  Defines IVF Frame Header type.
+  """
+
+  @typedoc """
+  IVF Frame Header.
+
+  Description of these fields is taken from:
+  https://chromium.googlesource.com/chromium/src/media/+/master/filters/ivf_parser.h
+
+  * `signature` - always "DKIF"
+  * `version` - should be 0
+  * `header_size` - size of header in bytes
+  * `fourcc` - codec FourCC (e.g, 'VP80'). 
+  For more information, see https://fourcc.org/codecs.php 
+  * `width` - width in pixels
+  * `height` - height in pixels
+  * `timebase_denum` - timebase denumerator
+  * `timebase_num` - timebase numerator. For example, if
+  `timebase_denum` is 30 and `timebase_num` is 2, the unit
+  of `ExWebRTC.Media.IVFFrame`'s timestamp is 2/30 seconds.
+  * `num_frames` - number of frames in a file
+  * `unused` - unused
+  """
+  @type t() :: %__MODULE__{
+          signature: binary(),
+          version: integer(),
+          header_size: integer(),
+          fourcc: integer(),
+          width: integer(),
+          height: integer(),
+          timebase_denum: integer(),
+          timebase_num: integer(),
+          num_frames: integer(),
+          unused: integer()
```

Comment:
`non_neg_integer()` would be a bit more precise. The same for `IVFFrame`'s fields.